### DISCUSSION TITLE
Bottom of 2x2 boxes or larger look horrible....

### DISCUSSION THREAD

**Discussion by timmeade** [kennetek/gridfinity-rebuilt-openscad#118] [Q&A]
Bambu P1P.  pla or ptg
![IMG_2003](https://github.com/kennetek/gridfinity-rebuilt-openscad/assets/1386476/0591b317-24b4-4085-b2c0-d32ff43f8340)

I have printed hundreds of GridFinity boxes over the last several months.  Most i start off with the other openscad version and they print fine.   

I was playing around with this verson and the code is much cleaner so i thought i
![IMG_2003](https://github.com/kennetek/gridfinity-rebuilt-openscad/assets/1386476/4f613022-4abe-4453-ac40-28f06052ffa0)
t would be a good base for what i want to add.

1x2 mag with label seem to print fine.  I also have been increasing the wall thickness to 1.8.

I tried a 2 X - 2Y 7H box and the transition from the 4 bases to the walls was horrible.   So i downloaded the latest from github and tried a totally clean version of the same.    No difference.   Here are some pictures.  Has anyone seen anything like this?

Thanks

Tim

**Comment by timmeade** [comment]
I forgot to add .4 nozzle .20 layer height these are PLA.

**Comment by Ruudjhuu** [comment]
I see some issues with your print. For starters, it seems like the corners are warping, so try to increase bed adhesion. Then the obvious issue: over extrusion on the transition of the base to the top par. This is where also the layers of the bottom of the bin are printed. My first thoughts are overall over extrusion. Which results in a bigger issue when printing solid layers? It is just a guess. I do not see this to be related to the openscad model. I would advise you to create a post in [FixMyPrint](https://www.reddit.com/r/FixMyPrint?utm_source=share&utm_medium=android_app&utm_name=androidcss&utm_term=1&utm_content=share_button)

**Reply by timmeade** [reply]
I would absolutely agree with you normally with the exception that this project is the only issue I am having.  Printer is Bambu labs p1p.  Every other print for months has been near perfect.  I will post the same box generated with other open scad project.  Near perfect.  Model looks the same on both in slicer

Very odd I admit.

**Comment by timmeade** [comment]
A little more info: I agree with @Ruudjhuu above that this SHOULD be the printer.    So in thinking about it i realized that these were all occurring at .20 layers, 35% infill and Triangles fill.

I switched to the generic pla .24 layer draft from bambu labs which has 15% infill and grid infill pattern.    Much much better.   See picture.   Trying .28 now.....

Documenting for the "Next" person.....

![IMG_2014](https://github.com/kennetek/gridfinity-rebuilt-openscad/assets/1386476/fe2e856e-30f3-47a2-abb4-cba21d780918)
